Rubberduck.Interaction/IMessageBox.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Forms = System.Windows.Forms;
+
+namespace Rubberduck.Interaction
+{
+    public interface IMessageBox
+    {
+        void Notify(string text);
+        void NotifyError(string text, string caption);
+        void NotifyWarn(string text, string caption);
+        bool Question(string text, string caption);
+        bool Prompt(string text, string caption);
+        bool ConfirmYN(string text, string caption);
+        bool ConfirmYN(string text, string caption, bool suggestion);
+        bool? Confirm(string text, string caption, bool? suggestion);
```

Comment:
`Confirm` isn't a Yes/No confirmation? It's best to be explicit - call it `ConfirmYesNoCancel` or something like that.